apps/web/__tests__/test/editor/blocks.cy.ts

```diff
@@ -3,14 +3,15 @@ import { EditorObject } from '../../support/pageObjects/EditorObject';
 
 describe('Blocks', () => {
     const editor = new EditorObject();
-  
+
     beforeEach(() => {
       cy.visitAndHydrate(paths.home);
     });
-  
+
     it('should ensure correct block logic and template', () => {
         editor.buttonsExistWithGroupClasses();
         editor.deleteBlock();
+        editor.addBlockTop();
+        editor.addBlockBottom();
```

Comment:
**suggestion**: Create a separate `it` for each block action. This makes it easier to see what's going on at a glance.

```
it('should delete a block', () => {})
it('should add a block above another block', () => {})
it('should delete a block below another block', () => {})
```

Reply:
This would make the test execution a lot slower. Let's talk when we are both here.

Reply:
We'll split it later.